docs/user/parameters.md

```
@@ -112,6 +112,16 @@ There are multiple use cases from the users perspective that dictate what parame
 | `--add-feature bmc` | Enable BMC feature | `--foreman-proxy-bmc` |
 | `--bmc-ipmi-implementation` | IPMI implementation to use for BMC | `--foreman-proxy-bmc-default-provider` |
 | `--bmc-redfish-verify-ssl` | Verify SSL certificates for Redfish BMC connections | `--foreman-proxy-bmc-redfish-verify-ssl` |
+| `--add-feature dns` | Enable DNS feauture | `--foreman-proxy-dns` |
+| `--dns-provider` | Select DNS provider | `--foreman-proxy-dns-provider` |
+| `--dns-server` | Configure the target DNS server | `--foreman-proxy-dns-server` |
+| `--dns-keyfile` | Configure the key for the target DNS server | `--foreman-proxy-keyfile` |
+
+##### Unmapped
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h4; Actual: h5

(MD001, heading-increment)